src/datatypes/external.ts

```diff
@@ -444,6 +444,19 @@ module Plywood {
       }
     }
 
+    static setCardinalityInflaterFactory(label: string): Inflater  {
+      return (d: any) => {
+        var v = d[label];
+        if ('' + v === "null") {
+          d[label] = null;
+          return;
+        }
+
+        if (typeof v === 'string') v = [v];
+        d[label] = v.length
```

Comment:
this could just be: `d[label] = Array.isArray(v) ? v.length : 1` this avoids creating a new array just to measure its length and is a better guard of v not being an array.